contrib/render-a11y-string/render-a11y-string.js

```diff
@@ -655,6 +655,10 @@ const handleObject = (
             break;
         }
 
+        case "html": {
+            throw new Error("KaTeX-a11y: html not implemented yet");
+        }
```

Comment:
The `\html*` commands wrap other commands.  I think we could probably call `buildString` on `tree.body`.  Do you see any issues with that approach?